.github/workflows/ci-tests.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         pip install tox
     - name: Run the tests
       run: |
-        tox -e py -- -n 2 test
+        tox -e py -- test
```

Comment:
@greg-finley, @tunetheweb: New experiment -- don't use parallelism for the test runs that also upload coverage reports. We know that coverage.py had issues with parallelism earlier this year. Perhaps they aren't completely resolved? 🤷